mono/metadata/marshal.c

```diff
@@ -691,7 +675,7 @@ mono_array_to_lparray (MonoArray *array)
 		nativeArray = (void **)g_malloc (sizeof(gpointer) * nativeArraySize);
```

Comment:
Consider `malloc0` so that `break` below does not leave uninitialized.
Or remove the `break`.